crates/bevy_color/src/oklcha.rs

```diff
@@ -191,9 +191,9 @@ impl Luminance for Oklcha {
 impl EuclideanDistance for Oklcha {
     #[inline]
     fn distance_squared(&self, other: &Self) -> f32 {
-        (self.lightness - other.lightness).powi(2)
-            + (self.chroma - other.chroma).powi(2)
-            + (self.hue - other.hue).powi(2)
+        FloatPow::squared(self.lightness - other.lightness)
+            + FloatPow::squared(self.chroma - other.chroma)
+            + FloatPow::squared(self.hue - other.hue)
```

Comment:
I think it'd be nicer to use `.squared()`. (I believe the type inferrence works here.)